### DISCUSSION TITLE
Jetson Mate | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#579] [Q&A] [answered]
# Jetson Mate | Seeed Studio Wiki

Jetson Mate

https://wiki.seeedstudio.com/Jetson-Mate/

**Comment by pramodhrachuri** [comment]
will this work with boards like orin nx and orin nano?

**Reply by xzxcessarr** [reply]
Hello, after testing we found it like not support for orin nx and orin nano now, the detail result we will inform in the forum and wiki after test all well

**Reply by pramodhrachuri** [reply]
Thanks!

**Reply by pramodhrachuri** [reply]
Any plans on making new boards to support them?

**Reply by xzxcessarr** [reply, marked as answer]
Hello, after test the board, we found the new version core board is not compatible with the old carrier board at hardware level, it may burn your core board

**Reply by pramodhrachuri** [reply]
Got it. Let me try to rephrase my question. Is seeed studio going to release a Jetson Mate V2 to support the new Jetson Orin modules?

Thanks for your prompt responses! And my apologies for the confusion

**Reply by xzxcessarr** [reply]
Hi, If you wanna get more detailed information about it, I suggest your to send an email to the fusion@seeed.io
